content/source/docs/enterprise-beta/saml/configuration.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: enterprise2
+page_title: "SAML Configuration - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-saml-configuration"
+---
+
+# Configuration
+
+Go to `https://[your-terraform-domain]/admin/settings/saml` and you’ll need to set the following values
+
+1. **Single Sign On URL**: specifies the HTTP(S) endpoint on your IdP for single sign-on requests. This value is provided by your IdP configuration.
+2. **Single Log Out URL**:  specifies the HTTP(s) endpoint on your IdP for single logout requests. This value is provided by your IdP configuration. Single Logut is not yet supported.
+3. **Identity Provider Certificate**: Specifies the X.509 Certificate as provided by the IdP configuration.
+4. **User email address**: (default: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`) Specifies the attribute to be used to identify the email (username) of the user.
+5. **Group Membership List**: (optional) Specifies the attributes used to identify team membership. This should be a comma-separated list.
```

Comment:
This is a lil unclear: Is this a comma-separated list of attributes, where the stored value of each attribute is a single team name? Or is it the name of an attribute, whose stored value is a comma-separated list of team names?